.github/workflows/pr.yaml

```
@@ -0,0 +1,24 @@
+# This is a template `pr.yaml` file for ops charms
+# This file is managed by bootstack-charms-spec and should not be modified
+# within individual charm repos. https://launchpad.net/bootstack-charms-spec
+
+name: PR workflow running lint checkers, unit and functional tests
+
+on:
+  push:
+    branches: [ master, main ]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches: [ master, main ]
+  pull_request_review:
+    types: [ submitted ]
+
+jobs:
+  pr:
+    uses: canonical/bootstack-actions/.github/workflows/pull-request.yaml@main
+    secrets: inherit
+    with:
+      python-version-unit: "['3.8', '3.10']"
+      python-version-func: "3.10"
+      tox-version: "<4"
+      commands: "['FUNC_ARGS=\"--series jammy\" make functional']"
```

We had tests for bionic, focal and jammy In previous `func_tests.yaml` workflow, so I think we should stick with that. 
```suggestion
      commands: "['FUNC_ARGS=\"--series bionic\" make functional', 'FUNC_ARGS=\"--series focal\" make functional', 'FUNC_ARGS=\"--series jammy\" make functional']"
```

---

I agree since we have a lot of bionic clouds